documentation/js/objects_shorthand.js

```diff
@@ -0,0 +1,14 @@
+var mask, name, output, pizzas, turtle, weapon;
+
+pizzas = 50;
+name = "Michelangelo";
+mask = "Orange";
+weapon = "Nunchuks";
+
+turtle = {
+  name: name,
+  mask: mask,
+  weapon: weapon
+};
+
+output = turtle.name + " wears an " + turtle.mask + " mask.";
```

Comment:
Please compile this file from objects_shorthand.coffee. Currently, it is out-of-sync.